pkg/storaged/dialog.jsx

```diff
@@ -1032,18 +1035,32 @@ export const TeardownMessage = (usage) => {
     if (!teardown)
         return null;
 
-    add_usage_message(parts, teardown.Mounts,
-                      _("This device has filesystems that are currently in use. Proceeding will unmount all filesystems on it."),
+    const uses = [];
+    add_usage_message(uses, teardown.Mounts,
+                      _("unmount"),
                       "Name", "MountPoint");
 
-    add_usage_message(parts, teardown.PhysicalVolumes,
-                      _("This device is currently used for volume groups. Proceeding will remove it from its volume groups."),
+    add_usage_message(uses, teardown.PhysicalVolumes,
+                      _("remove"),
                       "Name", "VGroup");
 
-    add_usage_message(parts, teardown.MDRaidMembers,
-                      _("This device is currently used for RAID devices. Proceeding will remove it from its RAID devices."),
+    add_usage_message(uses, teardown.MDRaidMembers,
+                      _("remove"),
                       "Name", "MDRaid");
 
+    if (uses.length > 0)
+        parts.push(
+            <div>
```

Comment:
Why `<div>`? Should this be `<>` (shorthand for `<React.Fragement>` https://reactjs.org/docs/fragments.html)